opentelemetry/proto/profiles/v1development/profiles.proto

```diff
@@ -119,6 +119,9 @@ message ProfilesDictionary {
 
   // Represents a mapping between Attribute Keys and Units.
   repeated AttributeUnit attribute_units = 7;
+
+  // Stacks referenced by samples via Sample.stack_index.
+  repeated Stack stack_table = 8;
```

Comment:
Maybe group stack_table with location_table and friends and keep attribute_units last since it's somewhat special in semantics?

Reply:
I guess with #672 it's less special .